src/routes/index.js

```diff
@@ -1,14 +1,49 @@
-import { Auth } from "../modules/auth/pages";
-import { Main } from "../modules/main/pages";
+import { Auth } from "../modules/auth/pages/index";
+import { Main } from "../modules/main/pages/index";
+import { Account } from "../modules/account/pages/index";
+import { Favorites } from "../modules/announcements/pages/favorites";
+import { Announcement } from "../modules/announcements/pages/announcement";
+import { Hotel } from "../modules/announcements/pages/hotel";
+import { Services } from "../modules/announcements/pages/services";
+import { Adoption } from "../modules/announcements/pages/adoption";
 
 export const routes = [
   {
     path: "/",
+    component: <Main />,
+    exact: "exact",
+  },
+  {
+    path: "/auth",
     component: <Auth />,
+  },
+  {
+    path: "/account",
+    component: <Account />,
+  },
+  {
+    path: "/announcements/hotel",
+    component: <Hotel />,
     exact: "exact",
   },
   {
-    path: "/main",
-    component: <Main />,
+    path: "/announcements/services",
+    component: <Services />,
+    exact: "exact",
```

Comment:
`exact` property of routing is rather used while it is necessary so the fact you use it almost whenever it is possible, shows that there is something not quite right with this approach 😄

Reply:
My mistake. I have copied those objects